src/%IPM/Main.cls

```diff
@@ -991,9 +991,9 @@ ClassMethod Init(ByRef pCommandInfo) [ Internal ]
 	Set tSCClassChanged = 0
 	
 	Set tPrimaryClass = ""
-	Set tCompositeSCClass = "%IPM.PackageManager.Developer.Extension.Composite"
-	Set tPackageMgrSCClass = "%IPM.PackageManager.Developer.Extension.PackageManager"
-	Set tSCInterface = "%IPM.PackageManager.Developer.Extension.SourceControl.Interface"
+	Set tCompositeSCClass = "%IPM.ExtensionBase.Composite"
+	Set tPackageMgrSCClass = "%IPM.Extension.ServerMenu.MenuItems"
```

Comment:
Yeah this class is for setting the package manager extension, it is used as the default extension when ipm is initializing, and secondary extension if another extension has been set primary. I think we can keep it since SetExtensionClass() can ignore non-existing class. But can also create another jira to completely remove the default logic from this class to perhaps an installer class of ipm-studio-ui?

Reply:
EDIT: Resolved now by adding super class %IPM.ExtensionBase.ServerMenu.Interface